src/states/Preroll.js

```diff
@@ -225,6 +233,24 @@ export default class Preroll extends AdState {
     }
   }
 
+  resumeAfterNoPreroll(player) {
+    player.ads.debug('**** contentResuming instead of direct transition');
+
+    // Resume to content and unblock play as there is no preroll ad
+    this.contentResuming = true;
+    player.ads._shouldBlockPlay = false;
+
+    player.ads.debug('paused?', player.paused(), player.ads._playRequested || player.ads._pausedOnContentupdate);
+    // Play the content if we had requested play or we paused on 'contentupdate'
+    // and we haven't played yet. This happens if there was no preroll or if it
+    // errored, timed out, etc. Otherwise snapshot restore would play.
+    if (player.paused() &&
+        (player.ads._playRequested || player.ads._pausedOnContentupdate)) {
+      player.ads.debug('Playing again since the player is paused');
```

Comment:
I thought it might be useful if we're debugging issues with resuming after nopreroll type situations. Thoughts?

Reply:
Yeah I'm not sure, my thinking is that it's pretty clear when there is a problem because the player will either play or not play, so you'd see it visually. Beyond that you'd need to go into the debugger anyway.